### DISCUSSION TITLE
How can I unbind copilot chat from Cmd-I?

### DISCUSSION THREAD

**Discussion by golergka** [community/community#58128] [Copilot Conversations]
### Select Topic Area

Question

### Body

I just installed Copilot Chat extension, but I can't use it because it overrides default Cmd-I behaviour — opening intellisense. I tried searching for this command in the command palette to change the shortcut, but I couldn't find it. How can I achieve this?

Also, product feedback: please don't override Cmd-I behaviour, it makes editor unusable.

**Comment by kwangcheol5** [comment]
To unbind the Cmd-I key from Copilot chat, you can modify the keybindings.json file in Visual Studio Code. Here are the steps to do so:

1. Open Visual Studio Code.
2. Click on the "Code" menu in the top menu bar.
3. Click on "Preferences", then "Keyboard Shortcuts".
4. In the search bar, type "copilot".
5. Find the "GitHub Copilot: Chat" command and click on the pencil icon next to it.
6. Press the backspace key to remove the current keybinding (Cmd-I).
7. Press Ctrl-S (Windows/Linux) or Cmd-S (Mac) to save the changes to the keybindings.json file.

After completing these steps, the Cmd-I key should no longer be bound to Copilot chat.

**Reply by johypark** [reply]
This shortcut was replaced to "Inline Chat: Start Code Chat".

**Reply by omarabualhija** [reply]
it's replaced a gain to  "Inline Chat: Start Inline Chat". 😅

my Device :mac
vs  version: 1.85.1 (Universal)

feel free to edit it if it is wrong

**Reply by oleksandr-kinship** [reply]
Now it's `Inline Chat: Open Inline Chat` :)

**Comment by omarabualhija** [comment]
1/ open  vs code 
2/in bottom left menu press on settings icon 
3/select keyboard shortcuts (or press commend +k + s for mac)
4/search on"Inline Chat: Start Inline Chat"

now u can edit as u want

my Device :mac
vs version: 1.85.1 (Universal)

**Comment by AyushB21** [comment]
1. Open Visual Studio Code.
2. Go to the "Code" menu on the top left corner (or the "File" menu on some operating systems).
3. Select "Preferences" and then "Keyboard Shortcuts."
4. In the Keyboard Shortcuts tab, you can search for specific commands or browse through the list to find the one you want to customize.
5. If you find the keybinding you want to change (such as Cmd+I for IntelliSense), you can right-click on it and choose "Change Keybinding" to assign a new key combination.

I hope this clarifies your question

**Comment by zykrkt** [comment]
1. Open Keyboard shortcuts
2. Search with `Inline Chat: Start in Editor`
3. Remove the keybinding `⌘+I` (Command + I)

**Comment by KScaesar** [comment]
Does **JetBrains** have a shortcut function similar to Ctrl + I (**Inline Chat: Start in Editor**) ?

**Comment by shersh1k** [comment]
Every vscode update makes me go and delete this bind, please stop it :)

**Comment by Isrealsix** [comment]
IT'S A VERY DUMB IDEA TO OVERRIDE USER'S EXISTING KEYBINDING WITH THEIR BS UPDATES. 

PS: ANYTIME I NEED SOMETHING TO FUEL MY ANGER, I REVISIT THIS BS FEATURE
